.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2.1
+
+executors:
+  golang:
+    docker:
+      - image: 'quay.io/influxdb/telegraf-ci:1.13.8'
+    working_directory: '/go/src/github.com/influxdata/telegraf'
+    environment:
+      GOFLAGS: -p=8
+  dockerizer:
+    docker:
+      - image: circleci/golang:1.14.4
+    environment:
+      IMAGE_NAME: filecoin/sentinel-drone
+      WIP_IMAGE_TAG: wip
+
+jobs:
+  test:
+    executor: golang
+    steps:
+      - checkout
+      - run: apt-get update
+      - run: apt-get install -y jq mesa-opencl-icd ocl-icd-opencl-dev
+      - restore_cache:
+          key: go-mod-v1-{{ checksum "go.sum" }}
+      - run: 'make deps'
+      - run: 'make tidy'
+      - save_cache:
+          name: 'go module cache'
+          key: go-mod-v1-{{ checksum "go.sum" }}
+          paths:
+            - '/go/pkg/mod'
+      - run: 'make'
+      - run: 'make check'
+      - run: 'make check-deps'
+      - run: 'make test'
+
+  docker-build:
+    executor: dockerizer
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: "18.09.3"
+      - run:
+          name: Build Docker image
+          command: |
+            docker build -t $IMAGE_NAME:$WIP_IMAGE_TAG .
+      - run:
+          name: Archive Docker image
+          command: docker save -o docker-image.tar $IMAGE_NAME
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./docker-image.tar
```

Comment:
Just out of curiousity, why is this valuable? I don't see it consumed by anything obvious.

Reply:
I was hedging my bets on how to automate publishing the image. This line is there so another CI job could push it to docker hub for a blessed branch, as we do in go-ipfs, but now we've got dockerhub watching the repo it's not needed. 

For context, we had issues with dockerhub builds failing for go-ipfs, so we moved the entire process into our CI. In this case, as we have a docker-build job that will tell us when the image can't be built, I'm taking a leap of faith, and trusting dockerhub again.

I'll remove this vestigial nub.